### PR TITLE
[ALLUXIO-3146] Fixed bin/alluxio copyDir failure on Mac

### PR DESCRIPTION
**Pull request by maboelhassan**
https://alluxio.atlassian.net/browse/ALLUXIO-3146

### CODE REVIEW

**Comment by maboelhassan** [comment]
https://alluxio.atlassian.net/browse/ALLUXIO-3146

File: bin/alluxio [hunk 1]
```diff
@@ -147,9 +147,16 @@ function copyDir {
     exit 1
   fi
 
+  ABS_PATH_COMMAND="readlink -f "$1""
```
Comment (aaudiber) [inline bin/alluxio]: does this work with double quotes inside of double quotes? Perhaps we could leave out the `$1` part and call `${ABS_PATH_COMMAND} $1` on line 159

Comment (maboelhassan) [reply bin/alluxio]: Yes, Both approaches work but I think yours is clearer so I changed it

**Comment by maboelhassan** [comment]
Applied feedback. Thanks @aaudiber

**Pull request merged**